SG-38470 Prevent QtImporter from raising an exception if QtWebEngine is not available (#1012)
shotgunsoftware/tk-core: docs/environment_variables.rst

----------------
Overrides the location where Toolkit stores data, which includes bootstrap data as well as bundle cache, cached thumbnails and other temp files.

``SHOTGUN_SKIP_QTWEBENGINEWIDGETS_IMPORT``
------------------------------------------
Use this variable to prevent Toolkit from importing the QtWebEngine modules when
importing the PySide2/PySide6 modules.

> This variable has been proven useful in odd case where a DCC would deadlock when

Reviewer: ```suggestion
This variable is useful when a DCC deadlocks while
```
I think we can simplify the sentence
